### PR TITLE
Fixes #33 - Added eslint options for all the web extensions

### LINKED ISSUE #33: Add eslint linting
We can probably just run this via npm scripts, so a package.json would be needed as well.

http://eslint.org/docs/user-guide/command-line-interface

We use the following settings for webcompat.com:

https://github.com/webcompat/webcompat.com/blob/master/package.json#L53

### PR DESCRIPTION
**Pull request by AkhilHector**
Signed-off-by: AkhilHector <akhilhector.1@gmail.com>

### CODE REVIEW

**Comment by AkhilHector** [comment]
- [ ] Updated gitignore
- [ ] Added eslint to package json
- [ ] Added `eslint` and `jpm` to devDependencies
- [ ] Test linted the `firefox-webext/background.js`

File: chrome/package.json [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    "type": "git",
+    "url": "git+ssh://git@github.com/webcompat/webcompat-reporter-extensions.git"
+  },
+  "bugs": {
+    "url": "https://github.com/webcompat/webcompat-reporter-extensions/issues"
+  },
+  "homepage": "https://github.com/webcompat/webcompat-reporter-extensions#readme",
+  "devDependencies": {
+    "jpm": "^1.1.4",
+    "eslint": "^1.8.0",
```
Comment (miketaylr) [inline chrome/package.json]: Can we go ahead and update to the latest eslint?

https://github.com/webcompat/webcompat.com/pull/1172 just updated webcompat.com to eslint 3.4 -- it also has the rule changes needed there.

**Comment by AkhilHector** [comment]
Sure @miketaylr

**Comment by miketaylr** [comment]
Hm, it seems like there's a lot of duplication here. I'm not sure what the best way to do this is... if we just have a single .eslintrc file that will make it so we don't have to update eslint options in multiple places.

**Comment by AkhilHector** [comment]
Yeah, i kind of had that in my mind. That's a good call ill create one eslintrc and use the same everywhere.

File: chrome/package.json (hunk 1 above)
Comment (AkhilHector) [inline chrome/package.json]: done, made the changes accordingly

**Comment by AkhilHector** [comment]
@miketaylr good to go

File: chrome/package.json (hunk 1 above)
Comment (miketaylr) [inline chrome/package.json]: Thanks!

(and thanks for being patient... traveling at the moment for a conference)

File: .eslintrc [hunk 2]
```diff
@@ -0,0 +1,46 @@
+{
+  "env": {
+    "browser": true
+  },
+  "extends": "eslint:recommended",
+  "globals": {
+    "jQuery": true,
```
Comment (miketaylr) [inline .eslintrc]: I should have caught this before... but these globals don't make sense for this repo. Can you remove and see what globals eslint complains about (and add those?).

File: chrome/package.json [hunk 3]
```diff
@@ -0,0 +1,25 @@
+{
+  "name": "chrome",
+  "version": "0.4.0",
+  "description": "Click to report an incompatible site at webcompat.com, and optionally upload a screenshot.",
+  "author": "Mike Taylor",
```
Comment (miketaylr) [inline chrome/package.json]: You don't have to change this, but at this point "Mike Taylor" as author doesn't make much sense. ^_^

Maybe something like "webcompat.com team"?

**Comment by AkhilHector** [comment]
Noted all the things, im starting to think why did i not recheck on the globals. Thanks for the heads up. Will make changes and submit today.

**Comment by miketaylr** [comment]
> im starting to think why did i not recheck on the globals

heh, i didn't even notice for the first 2 reviews either. 🙈 🙉

**Comment by miketaylr** [comment]
(sweet, just checking out the new review features)

File: chrome/background.js [hunk 4]
```diff
@@ -2,28 +2,28 @@
...
-  contexts: ["all"]
+  id: `webcompat-contextmenu`,
+  title: `Report site issue`,
+  contexts: [`all`]
 });
 
 
-function reportIssue (tab) {
-  chrome.tabs.captureVisibleTab({format: 'png'}, function(res) {
+function reportIssue() {
```
Comment (miketaylr) [inline chrome/background.js]: (I'm guessing maybe eslint complains? It's nice to know that a tab object is being passed into `reportIssue`, IMO.)

**Comment by miketaylr** [comment]
Looks good! Only one minor issue w/ including jpm and web-ext in non-Firefox package.jsons -- but I can push a follow up commit there.

**Pull request merged**